webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is outbound, the RTCOutboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		At the implementation's discretion, other stats that the
+		implementation considers relevant.
```

Comment:
A spec is a contract between the two sides of the API. Version skew will lead to one side that expects the old version to see behavior from the other side that is only specified in the new version. That's why it's nice to say what new behaviors can be expected in places like this.